test/corpus/interfaces.txt

```diff
@@ -170,7 +168,7 @@ public interface IGetNext<T> where T : IGetNext<T>
     name: (identifier)
     body: (declaration_list
       (method_declaration
-        type: (predefined_type)
+        returns: (predefined_type)
```

Comment:
@hvitved is this shift from type to returns going to cause upstream issues for semantic?

Reply:
I don't work on Semantic, but rather CodeQL. We don't currently have an official tree-sitter based C# CodeQL extractor, so nothing breaks for us (as long as we can still correctly parse what we could earlier).